recipes-firmware/firmware/firmware-qcom-boot-qcm6490_00005.0.bb

```diff
@@ -0,0 +1,16 @@
+DESCRIPTION = "QCOM NHLOS Firmware for Qualcomm Robotics RB3Gen2 platform"
```

Comment:
About the name of the recipe, do we expect custom boards to have a different set of boot firmware files? Just thinking if this should be really named qcm6490 or if we should instead have it named based on rb3gen2.

If we expect each custom board to have their own boot files, we should probably name this specifically to the target hardware.

Reply:
We do not anticipate custom boards having a different set of boot firmware files. Therefore, we believe naming the recipe based on qcm6490 is appropriate.